docs/README.md

```diff
@@ -361,9 +361,20 @@ These properties can be changed in the `blocks.json` file of your theme.
 | `pagination`        | `Enum`         | Pagination type (values: 'show-more' or 'infinite-scroll')                                                                           | `infinity-scroll` |
 | `mobileLayout`      | `MobileLayout` | Control mobile layout                                                                                                                | N/A               |
 | `showFacetQuantity` | `Boolean`      | If quantity of items filtered by facet should appear besides its name on `filter-navigator`                                          | `false`           |
+| `showFacetSearch`   | `Boolean`      | If true, a search input will be shown for each filter                                                                                | `false`           |
```

Comment:
Change it `facetSearchThreshold` of a type number, if the user doesnt want it to show, just leave it empty. If they want to always show, set it to zero, or 10 (or any other number) to show when there's a mininum of 10 options